### ISSUE TITLE
Guide to Open Data Licensing: next steps

### ISSUE THREAD

**Issue by pietercolpaert** [okfn/opendefinition#145]
I was impressed with the extensive explanation on data and intellectual property (IP) rights around the world at http://opendefinition.org/guide/data/. The page is an important resource to getting a global understanding of (the jungle of) data/content and IP.

However, in order for this page to be cited/linked by the many, we should be able to get rid of the disclaimer that this is not legal advise. Therefor, it would be interesting to call for legal experts in IPR to help out to validate the text for potential errors.

Furthermore, in order for this page to be seen as a scientific valuable resource, it seems interesting that, with the contributing authors, we submit this text to an open access journal.

Minor issues that might have been caused by jekyll:
- [x] References at the bottom are a bunch of empty `<li>` elements
- [x] The link to the source code at the bottom is a broken jekyll command
- [x] `A short 1-page instruction guide to applying an open data license can be found on the Open Data Commons site:` → shows an empty li element as well

**Comment by enyst** [comment]
Thank you for your feedback! Indeed the first issues you noted are broken refs, they exist in the source. The link to the instruction guide is missing, I think it should be this guide:
http://opendatacommons.org/guide/

I'll check tonight those references.

**Comment by AusGOAL** [comment]
The material listed under Australia is either old or incorrect and should
be removed immediately!

Kind regards

b

---

_Baden M Appleyard_
_Barrister_

_National Director - AusGOAL_
Mobile: +61(0)459 824 061
LinkedIn https://www.linkedin.com/in/badenappleyard

_AusGOAL_
Visit our Website http://www.ausgoal.gov.au | Like and Share us on
Facebook https://www.facebook.com/AusGOAL | Join our LinkedIn Group
http://linkd.in/oq5L3u | Follow Us on Twitter: @AusGOAL
https://twitter.com/#!/AusGOAL | +1 on Google Plus
https://plus.google.com/u/0/b/108169778015177255341/108169778015177255341/posts/p/pub
|
View our Slides https://docs.com/ausgoal
AusGOAL is the Government and Research Lead of Creative Commons Australia
The contents of this email may be subject to legal professional privilege.
Liability limited by a scheme approved under professional standards
legislation.

**Comment by mlinksva** [comment]
I don't think this page has been worked on much since it was [announced](http://blog.okfn.org/2007/06/27/guide-to-open-data-licensing/) in 2007. Too bad the original links are broken. Can this be fixed with redirects? For anyone who wishes to find bits that may be been lost in wiki->wordpress->jekyll migrations, see http://web.archive.org/web/*/https://okfn.org/wiki/OpenDataLicensing/

@pietercolpaert https://en.wikipedia.org/wiki/Legal_advice does not mean vetted by legal experts. I feel disclaimers are overused and may not be necessary here, but care very little.

**Comment by enyst** [comment]
@mlinksva thanks, I made a pass to fix things. Links seem to be working now without redirects:

http://enyst.github.io/opendefinition/guide/data/

**Comment by mlinksva** [comment]
Thanks @enyst, merged. I'll leave this open for further discussion of next steps.

**Comment by enyst** [comment]
I would suggest that an update of contents is among next steps. For one, the older science commons material it refers to is now the creative commons FAQ with updated info since 4.0 versions. I'm reviewing aspects like that. I have to check more things in depth.

I agree with @pietercolpaert that this is a very interesting resource. But if you wish to make it into a reference to be cited by many and/or vetted and/or submitted to a journal, there's a _lot_ more work to do on it.

**Comment by rufuspollock** [comment]
Great to see this being updated.

Re location / home for this: my suggestion would be a move to https://github.com/okfn/opendatahandbook somewhere appropriate (perhaps and appendix).

General point: let not the best be the enemy of the good ;-)

**Comment by pietercolpaert** [comment]
Just came across this report for Europe: http://ec.europa.eu/internal_market/copyright/docs/studies/1403_study2_en.pdf - seems like a very interesting resource to link to?

**Comment by BrucePerens** [comment]
Under the law of most US states, only an attorney who is admitted to the
state bar association and is directly contracted to you may render you
legal advice. Thus this must remain less than legal advice for the US. In
general it is overreaching for a web site to claim its content is actual
legal counsel, as we are not acquainted with the specifics of each user and
the user will not have the legal skill necessary to apply our data without
assistance.

I spend a lot of time cleaning up after do-it-yourself lawyering that goes
wrong. We should not lure our users into it.

Thanks

Bruce

On Oct 29, 2016 2:04 AM, "Pieter Colpaert" notifications@github.com wrote:

Just came across this report for Europe: http://ec.europa.eu/internal_
market/copyright/docs/studies/1403_study2_en.pdf - seems like a very
interesting resource to link to?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
https://github.com/okfn/opendefinition/issues/145#issuecomment-257080492,
or mute the thread

**Comment by mlinksva** [comment]
@pietercolpaert feel free to make a PR to add that link.

Another possibility: RDA-CODATA Legal Interoperability Interest Group. (2016). Legal Interoperability of Research Data: Principles and Implementation Guidelines. https://zenodo.org/record/162241
